### DISCUSSION TITLE
Fiona PyPI package has hit its wheel storage limit

### DISCUSSION THREAD

**Discussion by sgillies** [Toblerity/Fiona#1150] [Announcements]
Distributions for 1.9a3 failed to completely upload because we've hit PyPI's 10 GB storage limit. The issue is explained at https://pypi.org/help/#project-size-limit.

I plan to delete wheels for 1.6.x and 1.7.x pre-releases (a*, b*, and rc*) to make space, leaving source distributions in place. If you've pinned your projects to those versions, please make plans to move to newer releases or build your own wheels.

**Comment by EwoutH** [comment]
While old pre-release wheels probably won't disturb a lot of users, I would suggest also contacting PyPI to get the limit increased. Fiona is a big project with a long history but also [millions](https://pypistats.org/packages/fiona) of monthly downloads. The wheels being bigger due being a core package in the geospatial stack is also contributes. I think they would have no objections increasing the limit.

10 GB isn't that much and sooner or later we will encounter it again, especially if wheels get build for more platforms (macOS and Windows Arm64 for example).

It might also be time to drop Python 3.6 support, since it has been end-of-life for almost a year (since December 2021).

**Reply by adamjstewart** [reply]
I thought this was fixed in https://github.com/Toblerity/Fiona/issues/1151? A new limit was granted, and most of the missing wheels were uploaded. There is still at least one wheel missing though: https://github.com/Toblerity/Fiona/issues/1157.

**Reply by EwoutH** [reply]
Whoops didn’t see that, nevermind!
